cypress/integration/BarChart_spec.js

```diff
@@ -0,0 +1,72 @@
+describe('Dashboard View', () => {
+  const baseUrl = 'http://localhost:3000/'
```

Comment:
I think this cypress testing of the bar chart looks great! I'm doing something similar with the circle charts